### DISCUSSION TITLE
2.0.2-b1版本运行时候出现cmd界面提示，关闭时候主体程序UI也一并关闭，二次打开没有记忆上次打勾选项

### DISCUSSION THREAD

**Discussion by X012C** [cangzhang/ChampR#269] [General]
![image](https://github.com/cangzhang/champ-r/assets/99554281/ebe8e72a-436b-4a63-b019-2bb2bc7ed836)

**Comment by cangzhang** [comment]
在当前版本中(v2.0.2-b1)属于正常现象，后面会慢慢优化调整
